packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.jsx

```diff
@@ -983,11 +1008,13 @@ FinancialAssessment.propTypes = {
 export default connect(({ client, common, notifications }) => ({
     is_authentication_needed: client.is_authentication_needed,
     is_financial_account: client.is_financial_account,
-    is_svg: client.is_svg,
```

Comment:
Why did you remove `is_svg`?

Reply:
Trading assessment needs to be removed from Financial information for Mf client. So `!is_svg` was replaced by `!is_mf`

Reply:
TA or FA?
Also previously the form was removed for `svg` client should we add it to `mf` as well?
e.g. `!is_mf && !is_svg`?

Reply:
Incorporated the logic